funding/manager.go

```diff
@@ -2026,6 +2030,16 @@ func (f *Manager) handleFundingSigned(peer lnpeer.Peer,
 	// if we actually have the funding transaction.
 	if completeChan.ChanType.HasFundingTx() {
 		fundingTx := completeChan.FundingTxn
+
+		// Before we publish this transaction, we check that it won't
```

Comment:
Why do we need this here if it was already checked when we first got the request to reserve the funding reservation itself?

Reply:
Also, at this point the reservation has bee completed, and the existing funding manager state committed, so upon restart it'll attempt to rebroadcast the funding transaction as it's defined.

Reply:
Agreed, removed.